client/blocks/reader-full-post/_content.scss

```
@@ -8,15 +8,26 @@
 	overflow-wrap: break-word;
 	word-wrap: break-word;
 
-	h1 {
-		font-size: 28px;
+	h1,
+	h2,
+	h3,
+	h4,
+	h5,
+	h6 {
 		font-weight: 700;
+		margin-bottom: 15px;
+		margin-top: 30px;
+		line-height: 1.1;
+		text-align: left;
+	}
+
+	h1 {
+		font-size: 54px;
```

With the introduction of these: 

https://github.com/Automattic/wp-calypso/blob/a83f9acf2b30fc352da61669616f280df655134b/packages/typography/styles/variables.scss#L36

I thought I should offer this suggestion since there is a variable that matches the pixel value in the PR.

```suggestion
		font-size: $font-headline-large;
```